spec/parallel_tests/test/runtime_logger_spec.rb

```diff
@@ -10,62 +10,6 @@ def run_tests
     sh "ruby #{Bundler.root}/bin/parallel_test test -n 2"
   end
 
-  it "writes a correct log on test-unit" do
```

Comment:
Was dropping this intented? This is about test-unit, which seems supported? At least you have it on the Gemfile.